regression-test/src/app/calendar/page.tsx

```diff
@@ -30,9 +30,6 @@ const Calendar = cl as any
 export default function CalendarPage() {
   return (
     <main className="flex gap-8 p-8 flex-col items-start axe-test">
-      {/* Default config */}
-      <Calendar />
-
```

Comment:
unrelated regression test that was not needed because it just broke every time since it always displays the current date